cql/DashboardLibrary.cql

```diff
@@ -852,17 +984,33 @@ define Errors:
 
 // SUMMARY
 
+define Pregnant:
+  Exists(PregnancyDiagnoses) or
+  Exists(PregnancyObservations)
+
 // The Summary object represents the full Cervical Cancer Screening and Management Medical History to be displayed to the clinician. A robust UI
 // should be created in order to display this information in a Cervical Cancer Screening and Management Dashboard
 
 define CervicalCancerScreeningAndMedicalHistory:
 {
-  Age: PatientAge,
-  Conditions: PertinentConditionsSummary,
-  Observations: PertinentObservationsSummary,
-  Medications: PertinentMedicationsSummary,
-  Procedures: PertinentProceduresSummary,
-  DiagnosticReports: DiagnosticReportsSummary,
-  Encounters: PertinentEncountersSummary,
-  Immunizations: PertinentImmunizationsSummary
+  patientInfo: {
```

Comment:
It would be nice to add a test for this at some point to make sure all of the right data is being routed to the dashboard, but this doesn't necessarily have to be a part of this refactor. I'll add an issue to address this.

Reply:
Added a GitHub Issue: https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/issues/26.